src/docs/asciidoc/index.adoc

```diff
@@ -29,15 +29,30 @@ include::{snippets}/TxHistoryControllerTest/mustBeAbleToGetTransactionForSpecifi
 All requests need header: `Authorization: Bearer JWT`.
 Chain ID must be provided in the request path, e.g. 80001.
 
+Transaction types are:
+
+RESERVE_INVESTMENT - Refers to the investment placed into the campaign before it is finalized. Can be canceled.
+
+CANCEL_INVESTMENT - Refers to the cancellation of the reserved investments.
+
+COMPLETED_INVESTMENT - Refers to the investment claimed when the campaign is finalized.
+
+CREATE_PAYOUT - Refers to the payout manager transferring certain amount of tokens to its contract, making it
+eligible for individual addresses which invested into the asset to collect its share.
+
+REVENUE_SHARE - Refers to the investor addresses collecting revenue share.
+
```

Comment:
Put this inside section `Get transactions history`. 
Use ascii doc proper syntax for listing - https://asciidoctor.org/docs/asciidoc-writers-guide/#lists-of-things
```
.Possible transaction types are:
* RESERVE_INVESTMENT - Refers to the investment placed into the campaign before it is finalized. Can be cancelled.
* CANCEL_INVESTMENT - Refers to the cancellation of the reserved investments.
* COMPLETED_INVESTMENT - Refers to the investment claimed when the campaign is finalized.
* CREATE_PAYOUT - Refers to the payout manager transferring a certain amount of tokens to its contract, making it
eligible for individual addresses which invested into the asset to collect its share.
* REVENUE_SHARE - Refers to the investor addresses collecting revenue share.
```